components/header/ThemeSwitch.tsx

```
@@ -0,0 +1,41 @@
+
+export default function ThemeSwitch({ onChange, value }: { onChange: (theme: string) => void; value: string }) {
+  const [open, setOpen] = useState(false);
+  return (
+    <div className="relative">
+      <button
+        className="flex items-center gap-2 px-3 py-1 rounded border bg-white dark:bg-gray-800 hover:bg-gray-100 dark:hover:bg-gray-700 shadow"
+        onClick={() => setOpen((v) => !v)}
+        aria-label="Switch Theme"
+      >
+        <span className="font-medium">Theme</span>
+        <svg width="16" height="16" fill="none" viewBox="0 0 24 24"><path stroke="currentColor" strokeWidth="2" d="M6 9l6 6 6-6"/></svg>
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)